pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -781,7 +784,7 @@ func newKubeVirtConfigForCR(cr *hcov1beta1.HyperConverged, namespace string) *co
 	}
 	val, ok = os.LookupEnv("KVM_EMULATION")
 	if ok && val != "" {
-		cm.Data["debug.useEmulation"] = val
+		cm.Data[virtconfig.UseEmulationKey] = val
```

Comment:
Please also update the comments in lines 769 and 824 for consistency.

Reply:
Done